src/components/ProductCard/ProductCard.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import ProductCard from './ProductCard';
+import Tag from '../Tag/Tag';
+import Image from '../Image/Image';
+
+const meta = {
+  title: 'Components/Product card',
+  component: ProductCard,
+  argTypes: {
+    dropShadow: {
+      if: { arg: 'gradientBrand', truthy: false }
+    },
+    gradientBrand: {
+      if: { arg: 'dropShadow', truthy: false }
+    }
+  },
+  tags: ['autodocs']
+} satisfies Meta<typeof ProductCard>;
+
+export default meta;
+type Story = StoryObj<typeof ProductCard>;
+
+export const Playground: Story = {
+  args: {
+    texts: {
+      title: 'Product card title',
+      description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis nec vestibulum augue, viverra aliquet nunc. Cras eget felis sodales, vestibulum neque ac, rhoncus ipsum.'
+    },
+    linkTo: 'docs-demo-link',
+    headingLevel: 'h2'
+  }
+};
+
+export const SemanticCardTitle: Story = {
+  name: 'Semantic card title',
+  args: {
+    ...Playground.args
+  }
+};
+
+export const NonSemanticCardTitle: Story = {
+  name: 'Non-semantic card title',
+  args: {
+    ...Playground.args,
+    headingLevel: undefined,
```

Comment:
are we defining this because we're referencing playground and it's defined there? ideally we'd just want them to not specify headingLevel, so we may want to reconsider the reference to playground for this one

Reply:
I think this prop yelled at me when I used the previous way of defining the storyName, it's removed now.